server/routes/visitJourney/selectVisitors.ts

```diff
@@ -25,6 +25,13 @@ export default class SelectVisitors {
     }
     req.session.visitorList.visitors = visitorList
 
+    let adultVisitors = false
+    visitorList.forEach(visitor => {
```

Comment:
This will work fine but it will process all the visitors each time - even if the first one it finds is an adult. Admittedly I don't think this is going to be a big performance problem(!) but you could use something like `Array.some()` (see [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)).

So, (untested) perhaps something like:
```typescript
const adultVisitors = visitorList.some(visitor => visitor.adult === true)
```

(Also, minor point: as `adultVisitors` is a boolean, it might be clearer to name it something like `atLeastOneAdult` or `adultVisitorAvailable` - to avoid confusion with `adultVisitors` elsewhere in the `POST` route where it's used to hold an array of `VisitorListItem`

Reply:
Changed the implementation to include both of these suggestions, thanks!